services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/accessors/bsm/OdeBsmJsonRepositoryImpl.java

```diff
@@ -15,24 +16,23 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import us.dot.its.jpo.geojsonconverter.DateJsonMapper;
-import us.dot.its.jpo.ode.api.ConflictMonitorApiProperties;
 import us.dot.its.jpo.ode.model.OdeBsmData;
 
 @Component
 public class OdeBsmJsonRepositoryImpl implements OdeBsmJsonRepository {
 
     private final MongoTemplate mongoTemplate;
-    private final ConflictMonitorApiProperties props;
+
+    @Value("${maximumResponseSize}")
+    int maximumResponseSize;
```

Comment:
chore(blocking): together with the chore below. use constructor injection and keep this property private
```suggestion
    private final int maximumResponseSize;
```

Reply:
The maximumResponseSize variable is removed and no longer necessary, as this is handled by the controllers